docs/data/data-grid/editing/editing.md

```diff
@@ -540,6 +540,10 @@ Note that the `onCellEditStart` and `onCellEditStop` props also have to be used
 
 {{"demo": "LinkedFieldsCellEditing.js", "bg": "inline", "defaultCodeOpen": false}}
 
+### Usage with `@mui/x-date-pickers`
+
```

Comment:
Could you add a paragraph explaining what is this demo? It might obvious I agree, but visually is weird to start with the demo. You could link to https://mui.com/x/react-data-grid/editing/#create-your-own-edit-component and https://mui.com/x/react-data-grid/filtering/#create-a-custom-operator

Reply:
Done